ddtrace/contrib/falcon/middleware.py

```diff
@@ -50,7 +50,7 @@ def process_response(self, req, resp, resource, req_succeeded=None):
         if not span:
             return  # unexpected
 
-        status = httpx.normalize_status_code(resp.status)
+        status = resp.status.split(" ")[0]
```

Comment:
If we only want a single split we could either use `partition` or set the `maxsplit` argument to 1. I'd do something like:
~~~ python
status = resp.status.partition(" ")[0]
~~~
This will prevent scanning the string for further splitting points, but would still create 3 strings. Perhaps searching for the first space and returning a slice would be the most efficient approach.